packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.scss

```diff
@@ -132,8 +132,15 @@
 
       &:focus-visible {
         border-radius: var(--gse-ui-calendarMenu-month-focusBorderRadius);
-        outline: var(--gse-semantic-focusRing-default-border-width) solid
-          var(--gse-semantic-focusRing-default-border-color);
+        outline: var(
+            --gse-semantic-focusRing-default-border-width,
+            --gse-semantic-focusOutline-md-borderWidth
```

Comment:
the preferred token is 3px and the fallback is 2px. that correct?

Reply:
@VirtualClover is this change corect?